src/modules/sub-items/sub.items.module.js

```diff
@@ -344,13 +380,141 @@ export default class SubItemsModule extends Component {
+                        locationName: eZ.helpers.text.escapeHTML(location.ContentInfo.Content.Name),
+                        locationHref: this.props.generateLink(location.id),
+                    },
+                    'sub_items'
+                ),
+            };
+
+            window.eZ.helpers.notification.showSuccessNotification(message, () => {}, rawPlaceholdersMap);
+        }
+    }
+
+    afterBulkHide(successItems, failedItems) {
+        this.deselectAllItems();
+        this.discardActivePageItems();
+
+        this.toggleBulkOperationStatusState(false);
+
+        if (failedItems.length) {
+            const modalTableTitle = Translator.trans(
+                /*@Desc("Content items cannot be hidden (%itemsCount%)")*/
+                'bulk_hide.error.modal.table_title', {itemsCount: failedItems.length}, 'sub_items'
+            );
+            const notificationMessage = Translator.trans(
+                /*@Desc("%failedCount% of the %totalCount% selected item(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.")*/
+                'bulk_hide.error.message',
+                {
+                    failedCount: failedItems.length,
+                    totalCount: successItems.length + failedItems.length,
+                },
+                'sub_items'
+            );
+            const rawPlaceholdersMap = {
+                moreInformationLink: Translator.trans(
+                    /*@Desc("<u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>")*/
+                    'bulk_action.error.more_info', {}, 'sub_items'
+                ),
+            };
+
+            this.handleBulkOperationFailedNotification(failedItems, modalTableTitle, notificationMessage, rawPlaceholdersMap);
+        }
+
+        if (successItems.length) {
+            const message = Translator.trans(
+                /*@Desc("The selected location(s) have been hidden.")*/
+                'bulk_hide.success.message', {}, 'sub_items'
+            );
+            window.eZ.helpers.notification.showSuccessNotification(message);
+        }
+    }
+
+    afterBulkUnhide(successItems, failedItems) {
+        this.deselectAllItems();
+        this.discardActivePageItems();
+        this.toggleBulkOperationStatusState(false);
+
+        if (failedItems.length) {
+            const modalTableTitle = Translator.trans(
+                /*@Desc("Locations cannot be revealed (%itemsCount%)")*/
+                'bulk_unhide.error.modal.table_title', {itemsCount: failedItems.length}, 'sub_items'
+            );
+            const notificationMessage = Translator.trans(
+                /*@Desc("%failedCount% of the %totalCount% selected location(s) could not be revealed because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.")*/
```

Comment:
```suggestion
                /*@Desc("%failedCount% of the %totalCount% selected Location(s) could not be revealed because you do not have proper user permissions. {{ moreInformationLink }} Contact your Administrator to obtain permissions.")*/
```